src/components/pages/test-pages/radar-test-page/RadarTestPage.styles.ts

```diff
@@ -0,0 +1,21 @@
+import { SxProps, Theme } from '@mui/material/styles';
+
+export const wrapper: SxProps<Theme> = {
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  padding: '36px',
+  gap: '18px',
+  ul: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '8px',
+  },
+  backgroundColor: {
+    mobile: '#212121',
```

Comment:
colors from theme

Reply:
will be done later, when we have the palette